ViAn/GUI/videowidget.cpp

```diff
@@ -845,15 +846,45 @@ void VideoWidget::load_marked_video(VideoProject* vid_proj) {
     }
 }
 
-void VideoWidget::enable_video_btns() {
+void VideoWidget::remove_item(VideoProject* vid_proj) {
```

Comment:
Perhaps not necessary here but remember to add a comment head to all function with short description of what the function actually does.